net/prometheus/Portfile

```diff
@@ -0,0 +1,120 @@
+
+build.env           GOPATH=${workpath} \
+                    PATH=$env(PATH):${workpath}/bin
+
+build.target        build
+
+use_configure       no
+installs_libs       no
+use_parallel_build  no
+
+checksums \
+  rmd160    e066bbfce9ba96641f041adfa649a7a429a34856 \
+  sha256    0362f4aa2fb44cc2c572df140da742bdf99fe9f338157a83f6634694fd693000 \
+  size      11752395
+
+set prom_user       ${name}
+set prom_conf_dir   ${prefix}/etc/${name}
+set prom_conf_file  ${prom_conf_dir}/prometheus.yml
+set prom_data_dir   ${prefix}/var/db/${name}
+set prom_doc_dir    ${prefix}/share/doc/${name}
+set prom_share_dir  ${prefix}/share/${name}
+
+add_users           ${prom_user} \
+                    group=${prom_user} \
+                    realname=Prometheus
+
+post-extract {
+    copy  ${filespath}/org.macports.prometheus.plist \
+          ${workpath}/org.macports.prometheus.plist
+
+    reinplace "s|@NAME@|${name}|g" \
+        ${workpath}/org.macports.prometheus.plist
+
+    reinplace "s|@USER@|${prom_user}|g" \
+        ${workpath}/org.macports.prometheus.plist
+
+    reinplace "s|@GROUP@|${prom_user}|g" \
+        ${workpath}/org.macports.prometheus.plist
+
+    reinplace "s|@PREFIX@|${prefix}|g" \
+        ${workpath}/org.macports.prometheus.plist
+
+    reinplace "s|@CONF_FILE@|${prom_conf_file}|g" \
+        ${workpath}/org.macports.prometheus.plist
+
+    reinplace "s|@DATA_DIR@|${prom_data_dir}|g" \
+        ${workpath}/org.macports.prometheus.plist
+}
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/${name}  ${destroot}${prefix}/bin/${name}
+    xinstall -m 755 ${worksrcpath}/promtool ${destroot}${prefix}/bin/promtool
+
+    xinstall -d 755 ${destroot}${prom_conf_dir}
+    xinstall -d 755 ${destroot}${prom_doc_dir}
+    xinstall -d 755 ${destroot}${prom_share_dir}
+
+    xinstall -m 755 -o ${prom_user} -g ${prom_user} -d \
+        ${destroot}${prom_data_dir}
+
+    touch ${destroot}${prom_conf_dir}/.keep
+
+    copy ${filespath}/example-prometheus-config.yml \
+        ${destroot}${prom_share_dir}/
+
+    copy {*}[glob ${worksrcpath}/docs/*] ${destroot}${prom_doc_dir}/
+
+    xinstall -d -m 755 \
+        ${destroot}${prefix}/etc/LaunchDaemons/org.macports.${name}
+
+    xinstall -m 0644 -o root -W ${workpath} org.macports.${name}.plist \
+        ${destroot}${prefix}/etc/LaunchDaemons/org.macports.${name}
+
+    xinstall -d -m 755 ${destroot}/Library/LaunchDaemons
+
+    ln -s ${prefix}/etc/LaunchDaemons/org.macports.${name}/org.macports.${name}.plist \
+        ${destroot}/Library/LaunchDaemons/org.macports.${name}.plist
+
+    destroot.keepdirs-append ${destroot}${prom_data_dir}
+}
+
+post-activate {
+    if {![file exists ${prom_conf_file}]} {
+        copy  ${prom_share_dir}/example-prometheus-config.yml ${prom_conf_file}
+    }
+}
+
+notes "
+To change Prometheus configuration, you can make edits to:
+
+ ${prom_conf_file}
+
+To enable the Prometheus service, use `port load`, as follows:
+
+\$ sudo port load ${name}
```

Comment:
Please remove the blank lines in the `notes`.

Reply:
Certainly, but I feel like it might have a negative impact on readability.

Reply:
Well, I suppose it will be fine to keep them. We probably need to write guidelines for the notes field.